src/misc.jl

```diff
@@ -0,0 +1,39 @@
+# function to more accurately compute cos(x + xn)
+# when x >> xn
```

Comment:
this doesn't require x>>xn

Reply:
also, you should probably not name it `mycos`, maybe `cos_sum`?

Reply:
Sounds good! Meant more that cos(x + xn) is inaccurate when x >> xn not `mycos`. I will rename and remove that.

Reply:
the problem with `cos(x+xn)` isn't really that `x>>xn`, but that `abs(x+xn) >> pi/2`.